.github/workflows/download-cross-workflow.js

```diff
@@ -0,0 +1,14 @@
+// @noflow
+const fs = require("fs");
+const path = require("path");
+
+const actifactFolderPath = path.join(process.cwd());
```

Comment:
Since the `name` param is provided for `download-artifact` ([b](https://github.com/marilyn79218/react-lazy-show/pull/19/files#r648109251)), the artifact will be downloaded at `project_root_directory/${file_name}`.

Hence, we can download the file directly under current working directory `process.cwd()` instead of [this](https://github.com/marilyn79218/react-lazy-show/pull/20/files#diff-a557fb9f5dcb917977996fc7f760f4093b9680cd3df511de2fd130951910b33bR17) or [this](https://github.com/marilyn79218/react-lazy-show/pull/20/files#diff-a557fb9f5dcb917977996fc7f760f4093b9680cd3df511de2fd130951910b33bR22).